recipes-connectivity/networkmanager/networkmanager/firstboot-networkmanager-setup

```diff
@@ -28,6 +28,11 @@ done
 systemctl disable dnsmasq
 systemctl stop dnsmasq
 
+rm /etc/resolv.conf
```

Comment:
maybe for next release we could bring NM 1.6 that plays better with systemd-resolved so we wouldn't need resolveconf